src/test/kotlin/sirius/kernel/commons/StringsTest.kt

```diff
@@ -106,6 +106,17 @@ class StringsTest {
         assertNull(Strings.firstFilled(""))
     }
 
+    @Test
+    fun isHttpUrl() {
+        assertTrue(Strings.isHttpUrl("https://example.com"))
```

Comment:
Please provide more test cases
- with username & password
- with username, w/o password
- with path
- with port

Reply:
The more I think about it, the more I would prefer a `@ParameterizedTest` with `@CsvSource` (or the like) … as the individual asserts are mutually independent and should thus be considered separately, not consecutively.